sdk/search/search/README.md

```diff
@@ -400,6 +400,121 @@ for (const result of updateResult.results) {
 }
 ```
 
+### 3. Create and authenticate a `SearchServiceClient`
+
+You could use the following code to create and authenticate a `SearchServiceClient`. 
+
+```js
+import { SearchServiceClient, SearchApiKeyCredential }  from "@azure/search";
+
+const endPoint: string = "<endpoint>";
+const apiKey: string = "<apikey>";
+
+const credential: SearchApiKeyCredential = new SearchApiKeyCredential(apiKey);
+const client: SearchServiceClient = new SearchServiceClient(
+  endPoint,
+  credential
+);
+```
+
+#### Examples
+
+#### Get a list of existing indexes in the service
+
+```js
```

Comment:
it was asked to put importing and creating the client in every sample for easier copy/paste

Reply:
Done